server/db/add_client.sh

```diff
@@ -49,8 +50,9 @@ else
             echo "  Display Name: $unit"
```

Comment:
Same as in the add_buttons_to_client.sh file, please change line 48 from

```
    while IFS=",", read -r unit phone_number button_serial_number; do
```

To

```
    while IFS=",", read -r unit phone_number button_serial_number locationid; do
```

For the locationid variable to be populated

Reply:
done!